### PR TITLE
[!!!][TASK] Remove Support for PHP < 5.6

### PR DESCRIPTION
**Pull request by srotsch**
* Use ::class notation
* Use short array syntax
* Apply PSR-2 code formatting

### CODE REVIEW

File: Classes/Utility/IconUtility.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ class IconUtility
      */
     public static function getIconForRecord($table, array $row)
     {
```
Comment (tomasnorre) [inline Classes/Utility/IconUtility.php]: This is also refactored in my drop TYPO3 6.2 ticket.

File: Configuration/TCA/Overrides/tx_crawler_configuration.php [hunk 2]
```diff
@@ -3,5 +3,5 @@
 
 // Compatibility with 6.2
 if (\TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionNumberToInteger(\TYPO3\CMS\Core\Utility\VersionNumberUtility::getNumericTypo3Version()) < 7000000) {
```
Comment (tomasnorre) [inline Configuration/TCA/Overrides/tx_crawler_configuration.php]: Is also refactored in my drop TYPO3 6.2

File: ext_localconf.php [hunk 3]
```diff
@@ -1,34 +1,34 @@
 <?php
 defined('TYPO3_MODE') or die();
 
-$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['GLOBAL']['cliKeys']['crawler'] 			= array('EXT:crawler/cli/crawler_cli.php','_CLI_crawler');
-$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['GLOBAL']['cliKeys']['crawler_im'] 		= array('EXT:crawler/cli/crawler_im.php','_CLI_crawler');
-$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['GLOBAL']['cliKeys']['crawler_flush'] 	= array('EXT:crawler/cli/crawler_flush.php','_CLI_crawler');
-$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['GLOBAL']['cliKeys']['crawler_multiprocess'] 	= array('EXT:crawler/cli/crawler_multiprocess.php','_CLI_crawler');
+$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['GLOBAL']['cliKeys']['crawler'] 			= ['EXT:crawler/cli/crawler_cli.php','_CLI_crawler'];
```
Comment (tomasnorre) [inline ext_localconf.php]: Identation is a little funnny here

File: Classes/Utility/HookUtility.php [hunk 4]
```diff
@@ -40,24 +45,24 @@ class HookUtility
     public static function registerHooks($extKey)
     {
         $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['tslib/class.tslib_fe.php']['connectToDB']['tx_crawler'] =
-            'AOE\\Crawler\\Hooks\\TsfeHook->fe_init';
+            TsfeHook::class . '->fe_init';
```
[the remark this answers is not in the document]
Comment (srotsch) [reply Classes/Utility/HookUtility.php]: Looks a bit strange, indeed. But the core does it this way, e.g.

    'itemsProcFunc' => \TYPO3\CMS\Backend\View\BackendLayoutView::class . '->addBackendLayoutItems',

in typo3/sysext/core/Configuration/TCA/pages.php

File: ext_localconf.php (hunk 3 above)
Comment (srotsch) [reply ext_localconf.php]: Tabs ftw, will have a look

**Pull request merged**